app/src/main/java/org/mozilla/fenix/perf/ProfilerReusableComposable.kt

```diff
@@ -98,3 +106,30 @@ fun WaitForProfilerDialog(
         }
     }
 }
+
+/**
+ * Preview example of ProfilerLabeledRadioButton
+ */
+@Composable
+@LightDarkPreview
+fun ProfilerLabeledRadioButtonPreview() {
```

Comment:
```suggestion
private fun ProfilerLabeledRadioButtonPreview() {
```
Previews should just be a private function since we would never want to expose this. as a public API. Also, I don't think we necessarily need the comment. If you decide to keep the comment, we should surround `ProfilerLabeledRadioButton` with `[]` so you can click to go to the definition in Android Studio.